python-sdk/nuscenes/can_bus/tutorial.ipynb

```diff
@@ -88,7 +90,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "nusc_can.plot_message_data(scene_name, 'pose', 'accel', dimension=1)"
+    "message_name = 'pose'\n",
+    "key_name = 'accel'\n",
+    "nusc_can.plot_message_data(scene_name, message_name, key_name, dimension=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also retrieve the raw data and compare the wheel speeds to the vehicle speeds. Here we convert the wheel speed from rounds per minute to m/s and the vehicle speed from km/h to m/s. We can see that there is a small offset between the speeds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Retrieve raw data.\n",
+    "wheel_speed = nusc_can.get_messages(scene_name, 'zoe_veh_info')\n",
+    "wheel_speed = np.array([(m['utime'], m['FL_wheel_speed']) for m in wheel_speed])\n",
+    "\n",
+    "veh_speed = nusc_can.get_messages(scene_name, 'vehicle_monitor')\n",
+    "veh_speed = np.array([(m['utime'], m['vehicle_speed']) for m in veh_speed])\n",
+    "\n",
+    "# Convert to m/s.\n",
+    "radius = 0.328\n",
```

Comment:
where did the this radius come from?

Reply:
The number was actually wrong. Correct is 0.305m. Fixed it and added a comment.